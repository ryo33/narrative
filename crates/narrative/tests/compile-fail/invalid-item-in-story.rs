#[narrative::story("Story with invalid item")]
trait InvalidItemStory {
    // Type aliases are not allowed in stories
    type SomeType = String;

    #[step("When using type")]
    fn use_type(&self);
}

fn main() {}
