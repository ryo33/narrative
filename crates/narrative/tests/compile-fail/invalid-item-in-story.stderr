error: expected a step, trait, struct, enum, or let
 --> tests/compile-fail/invalid-item-in-story.rs:4:5
  |
4 |     type SomeType = String;
  |     ^^^^
